jsapp/js/components/processing/sidebar/sidebarSubmissionDetails.module.scss

```diff
@@ -1,6 +1,6 @@
 @use '~kobo-common/src/styles/colors';
 @use 'scss/sizes';
-@use './singleProcessingVariables';
+@use '../singleProcessingVariables';
 
 .mediaWrapper {
```

Comment:
Can we split this file into `sidebarSubmissionData.module.scss` and `sidebarSubmissionMedia.module.scss`? AFAIK they don't share any of the styles

Reply:
todo for later